hygieia-starter-kit/Dockerfile

```diff
@@ -27,18 +78,17 @@ COPY /api/* /hygieia/api/
 COPY /system/* /etc/systemd/system/
 COPY /collectors/* /hygieia/collectors/
 COPY /ui/httpd.conf /etc/httpd/conf/
-ADD /certs/* /hygieia/cacerts/
+#ADD /certs/* /hygieia/cacerts/
 ADD /properties-builder.sh /hygieia/
 
 # For adding certs
-RUN ls -1 /hygieia/cacerts | while read cert; do \
-        openssl x509 -outform der -in /hygieia/cacerts/$cert -out $cert.der; \
-        $JAVA_HOME/bin/keytool -import -alias $cert -keystore $JAVA_HOME/jre/lib/security/cacerts -trustcacerts -file $cert.der -storepass changeit -noprompt; \
-        rm $cert.der; \
-    done
-
-RUN systemctl enable mongod; \
-	systemctl enable hygieiaapp.service; \
+#RUN ls -1 /hygieia/cacerts | while read cert; do \
```

Comment:
should we control this certs setup using a feature flag?

Reply:
I'm not sure why i commented this out... there should be no harm running this if they don't add any additional certs... i'll uncomment and rerum just to make sure

Reply:
Updated...plz review again thanks